src/app/ui/overrides/_adf-info-drawer.scss

```diff
@@ -1,3 +1,16 @@
 .adf-info-drawer-layout {
     height: 100%;
-}
\ No newline at end of file
+}
+
+.adf-manage-versions-empty,
+.adf-manage-versions-no-permission {
+    margin: 24px;
+    color: rgba(0, 0, 0, 0.54);
+    text-align: justify;
+
+    & &-icon {
```

Comment:
double `& &`

Reply:
It looks strange, I know, but that was intentional. For more clarity, this is now changed :)